addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -669,6 +669,15 @@ export class OdooEditor extends EventTarget {
         this._columnUi.remove();
     }
 
+    resetContent(value = '<p><br></p>') {
+        this.editable.innerHTML = value;
+        this.sanitize();
+        this.historyStep(true);
+        // innerHTML could set the _toRollback to truty value. As we reseted the
+        // content, we know we do not want to rollback.
```

Comment:
```
// The unbreakable protection mechanism detects an anomaly and attempts
// to trigger a rollback when the content is reset using `innerHTML`.
// Prevent this rollback as it would otherwise revert the new content.
```